client/js/components/hyperdrive/index.js

```diff
@@ -1,24 +1,20 @@
-var hyperdriveRenderer
-var noop = function () {}
-
-if (module.parent) {
-  hyperdriveRenderer = require('./../../app.js').getServerComponent('hyperdrive')
-} else {
-  hyperdriveRenderer = require('./client.js')
-}
+const hyperdriveRenderer = require('./client.js')
+const noop = function () {}
+var onclick = null
 
 module.exports = function (state, prev, send) {
   if (!module.parent && !state.archive.instance && state.archive.key) {
     send('archive:load', state.archive.key)
-  }
-  var onclick = (ev, entry) => {
-    if (entry.type === 'directory') {
-      send('archive:update', {root: entry.name})
-      return true
-    } else {
-      send('preview:file', {archiveKey: state.archive.key, entryName: entry.name}, noop)
-      return false
+    onclick = (ev, entry) => {
```

Comment:
now this isn't merging because i slightly changed this so that it sends `entry: entry` in line 13.